presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -281,7 +281,7 @@ private static long stringPositionFromEnd(Slice string, Slice substring, long in
         }
         while (foundInstances < instance);
 
-        return countCodePoints(string, 0, index) + 1;
+        return index + 1;
```

Comment:
index is already calculated with lastIndexOf of String type, doe not need to call countCodePoints again.